.github/workflows/ci.yml

```
@@ -499,10 +501,86 @@ jobs:
     name: '[Manual] Build Puppeteer latest'
     needs: manual
     runs-on: ubuntu-latest
+    environment: CI
     steps:
-    - uses: actions/checkout@v4
-    - name: puppeteer-latest
-      run: docker build . --file ./puppeteer/latest.Dockerfile --tag "puppeteer-latest:$(date +%s)"
+    - name: Check out the repo
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)